.rubocop_todo.yml

```diff
@@ -0,0 +1,110 @@
+# This configuration was generated by
```

Comment:
there is no need to use `.rubocop_todo.yml` in new project. Main purpose for `.rubocop_todo.yml` is to make it easier to add rubocop in huge projects with many violations. Since you are starting almost from scratch, I would recommend to get rid of this file and do rubocop autofix instead (`rubocop -a` )

Reply:
removed .rubocop_todo.yml from project source, form now on I won't use it in my fresh new projects